src/Altinn.AccessManagement.Core/Configuration/GeneralSettings.cs

```diff
@@ -24,5 +24,21 @@ public class GeneralSettings
         /// Open Id Connect Well known endpoint. Related to JSON WEB token validation.
         /// </summary>
         public string OpenIdWellKnownEndpoint { get; set; }
+
+        /// <summary>
+        /// Gets the SBL base adress from kubernetes environment variables and appsettings if environment variable is not set
+        /// </summary>
+        public string GetSBLBaseAdress
+        {
+            get
+            {
+                return Environment.GetEnvironmentVariable("GeneralSettings__SBLBaseAdress") ?? SBLBaseAdress;
+            }
+        }
+
+        /// <summary>
+        /// Gets or sets the host name.
+        /// </summary>
+        public string HostName { get; set; }
```

Comment:
The HostName of what? Should be more specifically named and described

Reply:
changed the name to AccessManagementApplicationHostName